src/celeritas/em/xs/MuBremsDiffXsCalculator.hh

```diff
@@ -39,7 +39,7 @@ namespace celeritas
  * The contribution to the cross section from the nucleus is given by
  * \f[
    \Phi_n = \ln \frac{B Z^{-1/3} (\mu + \delta(D'_n \sqrt{e} - 2))}{D'_n (m +
-   \delta \sqrt{e} B Z^{-1/3})} \ ,
+   \delta \sqrt{e} B Z^{-1/3})} ,
```

Comment:
Okay, but this is nothing to do with this MR.

Reply:
I can revert it. Seemed an overkill to fix it on a separate one.

Reply:
Yeah, too small unless it were bundled with some other doc fixes... I think you can keep it here.

Reply:
Isn't this ok as is though? `\ ` in latex is how to add a space between an equation and a punctuation.

Reply:
Yeah, that would be just an extra space when rendered. I can revert my nitpicking.